modules/ROOT/pages/endpoint-as2-send.adoc

```diff
@@ -8,16 +8,24 @@ Create the AS2 identifiers for your organization and your trading partners in th
 |===
 |Setting |Description |Required
 
-| Owner
-| The trading partner or host that owns the endpoint.
-| Yes
+|Owner
+|The trading partner or host that owns the endpoint.
+|Yes
 
-| Protocol
-| The protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window.
-| Yes
+|Usage
+|How the endpoint is used. For receive endpoints, the value in this field is `Send to partners`. The value in this field is automatically generated based on the type of endpoint created.
+|No
+
+|Protocol
+|The protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window.
+|Yes
+
+|Partner
+|The partner to whom the transaction is sent.
```

Comment:
Have we consistently treated "partner" as a "who" or a "what"? Standardize throughout.

Reply:
It seems to be a "who".